po/pt_BR.po

```diff
@@ -1660,64 +1643,55 @@ msgid "Bunker Blaster"
 msgstr "Arrasa-Defesas"
 
 #: data/base/messages/strings/resstrings.txt:31
-#: data/mp/messages/strings/resstrings.txt:1
-#: po/custom/fromJson.txt:378
+#: data/mp/messages/strings/resstrings.txt:1 po/custom/fromJson.txt:378
```

Comment:
There are _many_ lines with a change like this (removing a newline). Is it Poedit 2.2.1 doing it?

Reply:
> There are _many_ lines with a change like this (removing a newline).

PO is no linewise format and source reference comments are ignored.

> Is it Poedit 2.2.1 doing it?

Poedit support stated that it does not (they were not pleased I asked).
So that was done by the translator, maybe while using another editor.

Reply:
> > There are _many_ lines with a change like this (removing a newline).
> 
> PO is no linewise format and source reference comments are ignored.
> 

Right, but it makes it much more cumbersome to review the diff. (Which, admittedly, is not the ideal way of reviewing changes to a .po file - for multiple reasons.)

Reply:
> > > There are _many_ lines with a change like this (removing a newline).
> > 
> > 
> > PO is no linewise format and source reference comments are ignored.
> 
> Right, but it makes it much more cumbersome to review the diff. (Which, admittedly, is not the ideal way of reviewing changes to a .po file - for multiple reasons.)

I have removed this annoyance.
Unless you understand Portuguese, there is little left to review.